packages/connector-aliyun-dm/src/constant.ts

```diff
@@ -0,0 +1,29 @@
+import path from 'path';
+
+import { ConnectorType, ConnectorMetadata } from '@logto/connector-types';
+import { getFileContents } from '@logto/shared';
+
+// eslint-disable-next-line unicorn/prefer-module
+const currentPath = __dirname;
+const pathToReadmeFile = path.join(currentPath, 'README.md');
+const pathToConfigTemplate = path.join(currentPath, 'config-template.md');
+const readmeContentFallback = 'Please check README.md file directory.';
+const configTemplateFallback = 'Please check config-template.md file directory.';
+
+export const defaultMetadata: ConnectorMetadata = {
+  id: 'aliyun-dm',
+  type: ConnectorType.Email,
+  name: {
+    en: 'Aliyun Direct Mail',
+    'zh-CN': '阿里云邮件推送',
+  },
+  // TODO: add the real logo URL (LOG-1823)
```

Comment:
i would suggest to remove this line since we already have an issue to track this